clients/storagegrpc/storage_client.go

```diff
@@ -0,0 +1,46 @@
+package storagegrpc
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/forta-network/forta-core-go/protocol"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+)
+
+// Client allows us to communicate with an agent.
+type Client struct{}
+
+// NewClient creates a new client.
+func NewClient() *Client {
+	return &Client{}
+}
+
+// Dial dials an agent using the config.
+func Dial(serverURL string) (protocol.StorageClient, error) {
```

Comment:
Maybe a `DialWithContext` with a backoff instead of loop so that it's easier to handle cancellations?

Reply:
Thanks, I changed this to use `grpc.DialContext()` instead and let it pass along the context.

I also gave a thought to using backoff but I think backoff especially more useful whenever there's a possibility of outage in a remote dependency so we react quick enough and start waiting with longer periods. This dialing is targeted to a container within the same cluster that is expected to immediately work and I think the simple loop does the same job with lower compexity.